tests/smoke_tests/test_managed_job.py

```diff
@@ -105,19 +105,19 @@ def test_job_pipeline(generic_cloud: str):
         [
             f'sky jobs launch -n {name} tests/test_yamls/pipeline.yaml --cloud {generic_cloud} -y -d',
             'sleep 5',
-            f'{smoke_tests_utils.GET_JOB_QUEUE} | grep {name} | head -n1 | grep "STARTING\|RUNNING"',
+            rf'{smoke_tests_utils.GET_JOB_QUEUE} | grep {name} | head -n1 | grep "STARTING\|RUNNING"',
```

Comment:
IIRC, `rf` does not work well with some python versions, might worth searching for it a bit.

Reply:
That's interesting. Do you recall what Python versions you had in mind? Raw f-strings `rf''` were introduced very early along with [PEP 498](https://peps.python.org/pep-0498/#raw-f-strings) in Python 3.6, so any CPython version from 3.6 onward should support them.

[Python 3.12](https://docs.python.org/3/whatsnew/3.12.html#pep-701-syntactic-formalization-of-f-strings) did introduce some improvements to f-strings, like allowing backslashes inside expressions and supporting the same type of quotes inside the expressions, but those changes didn’t affect raw f-strings specifically.